docker/config/datavault.properties

```diff
@@ -129,6 +129,28 @@ ldap.searchFilter = uid
 # A list of the attributes you want to retrieve from LDAP
 ldap.attrs = attr1,attr2,etc
 
+# ArchiveStore settings
+# =====================
+# only ArchiveStores with the `enabled` property set to `true` will be initialized
+
+# Local File System
+archivestore.file.storageClass = org.datavaultplatform.common.storage.impl.LocalFileSystem
+archivestore.file.label = Default archive store (Local)
+archivestore.file.retrieveEnabled = true
+archivestore.file.enabled = false
```

Comment:
Should we not use env. variables for the `enabled` values?
And maybe `retrieveEnabled` as well?
Ideally we would put this in a Array: `export enabledArchives=("file" "s3" "tms")` but I don't think we would be able to use it in the `.properties` file.
That's just a thought if that's not possible or to complex I'm ok with the way it works at the moment...